app/models/concerns/waste_exemptions_engine/can_have_registration_attributes.rb

```diff
@@ -15,6 +15,10 @@ module CanHaveRegistrationAttributes
 
     # Some business types should not have a company_no
     def company_no_required?
+      llp_or_ltd?
```

Comment:
Is this change so you can use `llp_or_ltd?` as an alias for `company_no_required?`

Reply:
yep - haven't aliased `company_no_required` incase that were to change

Reply:
`company_no_required?` is used elsewhere in the codebase to make this determination. It's not clear that adding another term for the same thing helps. I think the key information here is not whether the business is ltd or llp, but whether the user journey needs to deal with companies house number, so `company_no_required?` is probably a reasonable name to use. But clearly you didn't find it intuitive, so it's not as intuitive as I thought - maybe need to discuss this :-)

Reply:
I've gone only by the ticket's requirement which is "only make this change if it's a LTD or LLP" - if that's synonymous with needing a company number then happy to rollback just give the word 👍

Reply:
Yes, that's correct, "company number is required" and "business type is LTD or LLP" are the same thing.